### PR TITLE
fixed json case + added episode ids + empty response error

### PR DESCRIPTION
**Pull request by ekleop**
it works really bad(only detection by file is available) without this fixes.

### CODE REVIEW

**Comment by JamsRepos** [comment]
Been working to test these new changes and full functionality has now been restored. Without this update, Shows will not scrobble past Season 1.

File: Jellyfin.Plugin.Simkl/Jellyfin.Plugin.Simkl.csproj
Comment (crobibero) [inline Jellyfin.Plugin.Simkl/Jellyfin.Plugin.Simkl.csproj:1]: This file's changes should be reverted

**Review by crobibero — CHANGES_REQUESTED** [review]

File: Jellyfin.Plugin.Simkl/Jellyfin.Plugin.Simkl.csproj
Comment (ekleop) [reply Jellyfin.Plugin.Simkl/Jellyfin.Plugin.Simkl.csproj:1]: done, then the version will be the same 3.0.0

File: Jellyfin.Plugin.Simkl/API/SimklApi.cs [hunk 1]
```diff
@@ -287,7 +298,11 @@ private static SimklHistory CreateHistoryFromItem(BaseItemDto item)
 
             var responseMessage = await _httpClientFactory.CreateClient(NamedClient.Default)
                 .SendAsync(options);
-            return await responseMessage.Content.ReadFromJsonAsync<T1>(_jsonSerializerOptions);
+            var jsopt = new JsonSerializerOptions
+            {
+                PropertyNameCaseInsensitive = true
+            };
```
[the remark this answers is not in the document]
Comment (ekleop) [reply Jellyfin.Plugin.Simkl/API/SimklApi.cs]: ok, didn't take into account the performance issue. Fixed.

**Review by crobibero — APPROVED** [review]

**Pull request merged**